decidim-core/app/helpers/decidim/cells_helper.rb

```diff
@@ -40,7 +44,7 @@ def withdrawable?
 
     def flagable?
       return unless from_context
-      return unless proposals_controller? || collaborative_drafts_controller?
+      return unless proposals_controller? || collaborative_drafts_controller? || meetings_controller?
```

Comment:
maybe we could refactor this approach with a `flagabble_controller?` method added to those controllers:
```ruby
return unless try(:flaggable_controller?)
```
It would remove references to Meetings and Proposals from the Core module and it would allow other modules to use this feature.

Reply:
It makes perfect sense, done!